frontend/lib/about/dev_settings_view.dart

```diff
@@ -166,6 +166,7 @@ class DevSettingsView extends StatelessWidget {
     );
   }
 
+  // You can either copy/paste from clipboard on ios or type 'adb shell input text <text>' in terminall for android
```

Comment:
I think this only applies for macOS. On linux I didn't have any problems with sharing the clipboard with the host system. @ztefanie @steffenkleinle ?
```sh
adb shell input text <text>
```

Reply:
Yes this is not needed on linux. Also I think this comment is unnecessary, as how to share code between hostsystem and emulator is not specific to our code and can be googled easily. On browserstack there is a special hack to copy from clipbord to emulators clipbord and maybe behaviour changes on ios / android / linux whatever in the future and we would have to update our code then.